requirements.txt

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 boto3>=1.35.96
-click>=8.1.7,<9.0.0
```

Comment:
This is correct change overall but probably doesn't belong here. (But I will not block on this.)